Dockerfile

```diff
@@ -1,27 +1,23 @@
-FROM alpine:3.6
+FROM alpine:3.7
 
-LABEL org.label-schema.name="docker-bench-security" \
-      org.label-schema.url="https://dockerbench.com" \
-      org.label-schema.vcs-url="https://github.com/docker/docker-bench-security.git"
+LABEL \
+  org.label-schema.name="docker-bench-security" \
+  org.label-schema.url="https://dockerbench.com" \
+  org.label-schema.vcs-url="https://github.com/docker/docker-bench-security.git"
 
-# Switch to the HTTPS endpoint for the apk repositories as per https://github.com/gliderlabs/docker-alpine/issues/184
-RUN sed -i 's/http\:\/\/dl-cdn.alpinelinux.org/https\:\/\/alpine.global.ssl.fastly.net/g' /etc/apk/repositories
+COPY ./*.sh /usr/local/bin/
+COPY ./tests/*.sh /usr/local/bin/tests/
 
+# Switch to the HTTPS endpoint for the apk repositories
+# https://github.com/gliderlabs/docker-alpine/issues/184
 RUN \
-  apk upgrade --no-cache && \
+  sed -i 's/http\:\/\/dl-cdn.alpinelinux.org/https\:\/\/alpine.global.ssl.fastly.net/g' /etc/apk/repositories && \
   apk add --no-cache \
     docker \
     dumb-init && \
-  rm -rf /usr/bin/docker-* /usr/bin/dockerd && \
-  mkdir /usr/local/bin/tests
-
-COPY ./*.sh /usr/local/bin/
-
-COPY ./tests/*.sh /usr/local/bin/tests/
+  rm -rf /usr/bin/docker?*
```

Comment:
Not really sure about this one.
```
$ ls -1 /usr/bin/docker?*
/usr/bin/docker-containerd
/usr/bin/docker-containerd-ctr
/usr/bin/docker-containerd-shim
/usr/bin/dockerd
/usr/bin/docker-init
/usr/bin/docker-proxy
/usr/bin/docker-runc
$ sudo rm -rf /usr/bin/docker?*
$ ls -1 /usr/bin/docker?*
ls: cannot access '/usr/bin/docker?*': No such file or directory
$
```

Reply:
hey @konstruktoid,

yeah thats purpose :)
so with `rm -rf /usr/bin/docker?*` you deleted all that stuff.
when doing a `ls -l /usr/bin/docker*` and there should be only docker be left.

cheers maik

Reply:
ah yes, my bad.